src/com/CommunicateMesh.cpp

```diff
@@ -226,10 +225,11 @@ void CommunicateMesh::broadcastSendMesh(const mesh::Mesh &mesh)
       _communication->broadcast(edgeIDs);
 
       std::vector<int> triangleIDs(numberOfTriangles * 3);
+      const auto &     meshTriangles = mesh.triangles();
       for (int i = 0; i < numberOfTriangles; i++) {
-        triangleIDs[i * 3]     = mesh.triangles()[i].edge(0).getID();
-        triangleIDs[i * 3 + 1] = mesh.triangles()[i].edge(1).getID();
-        triangleIDs[i * 3 + 2] = mesh.triangles()[i].edge(2).getID();
+        triangleIDs[i * 3]     = meshTriangles[i].edge(0).getID();
+        triangleIDs[i * 3 + 1] = meshTriangles[i].edge(1).getID();
+        triangleIDs[i * 3 + 2] = meshTriangles[i].edge(2).getID();
```

Comment:
Same as above for `3`.

Reply:
See https://github.com/precice/precice/pull/1008#discussion_r624929570